ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -424,6 +424,29 @@ if [[ "${TEST_INSTALL:-}" = "yes" ]]; then
     /bin/false
   fi
 
+  io::log_yellow "Verify no extraneous files were installed."
+  # This script is only run on a Fedora-based build, so it's fine that this
+  # setting is not portable.
+  export PKG_CONFIG_PATH="/var/tmp/staging/lib64/pkgconfig:/usr/local/lib64/pkgconfig"
```

Comment:
This script is run from `ubuntu` images also, for example the `clang-3.8` build 

https://github.com/googleapis/google-cloud-cpp/blob/4157e16a81c8a2c8a56af16c1e3f1f5b96a46bb9/ci/kokoro/docker/build.sh#L212-L219

Would this setting become more portable if we simply pre-pended the desired path to whatever this environment variable already was? Like:

```shell
export PKG_CONFIG_PATH="/var/tmp/staging/lib64/pkgconfig:${PKG_CONFIG_PATH:-}"
```

WDYT?

Reply:
See my comment above in response to Todd. I checked the kokoro logs for the `cmake-super` build as well. With that being I said, I do think the code is an improvement so I added it. We'll need to make similar changes to `check-abi.sh` since that's where I copied the code from.

Reply:
SGTM. Sorry for missing that you and @mr-salty had already discussed this.